Fix typos and small issues with the tutorial

stefsmeets opened the pull request:
# Pull Request checklist

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [x] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [x] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [ ] Give this pull request a descriptive title.
- [ ] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [ ] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [ ] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Hi guys, just creating this pull request to keep track of some small issues I'm running into with the tutorial. It's mostly typos, formatting errors, and some clarifications here and there.

- Closes #123 
- Closes #122

SarahAlidoost commented on _episodes/02-installation.md:
```diff
@@ -40,6 +40,10 @@ documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/install
     should work on any recent system. If you have problems in the next step(s)
     you can alternatively try a 32 bit version.
```
Please replace all of the text before the code block below with "Please download Miniconda3 by running"

SarahAlidoost commented on _episodes/02-installation.md:
```diff
@@ -135,10 +139,10 @@ Next, download the file
 page](https://julialang.org/downloads/). Download the file directly to the
-`~/julia` directory or move it there after downloading. To extract the file, you
-can use the following command:
+`~/julia` directory or move it there after downloading. To download and extract the file, you can use the following commands:
```
```suggestion
Next, to download and extract the file `julia-1.0.5-linux-x86_64.tar.gz`, you can use the following commands:
```

SarahAlidoost commented on _episodes/02-installation.md:
```diff
@@ -135,10 +139,10 @@ Next, download the file
 [`julia-1.0.5-linux-x86_64.tar.gz`](https://julialang-s3.julialang.org/bin/linux/x64/1.0/julia-1.0.5-linux-x86_64.tar.gz)
 by clicking the link or going to the [Julia downloads
 page](https://julialang.org/downloads/). Download the file directly to the
```
Please remove this paragraph, no need to have three different instructions for downloading the file.

SarahAlidoost commented on _episodes/04-recipe.md:
```diff
@@ -398,16 +398,20 @@ Exemplary output (depending on the directory paths and package versions that are
 > ## Your output plot(s).
-> Plot for the dataset(s):
->
+> Plots for the dataset(s) are located in `./recipe_example_#_#/diag_timeseries_temperature/timeseries_diag/`
```
```suggestion
> Plots for the dataset(s) are located in `./recipe_example_#_#/plots/diag_timeseries_temperature/timeseries_diag/`
```

SarahAlidoost commented on _episodes/06-debugging.md:
```diff
@@ -231,7 +231,7 @@ config_developer_file: {HOME}/.esmvaltool/config_reference.yml
 > ## ESMValTool can’t locate the data
 >
 > You are assisting a colleague with ESMValTool. The colleague changes the
```
```suggestion
> You are assisting a colleague with ESMValTool. The colleague replaces the
```

SarahAlidoost commented on _episodes/06-debugging.md:
```diff
@@ -231,7 +231,7 @@ config_developer_file: {HOME}/.esmvaltool/config_reference.yml
 >
 > You are assisting a colleague with ESMValTool. The colleague changes the
-> ``project`` entry to ``CMIP6`` and runs the recipe. However, ESMValTool
+> ``datasets/project`` entry to ``CMIP6`` and runs the recipe. However, ESMValTool
```
```suggestion
> "CMIP5" in ``project: CMIP5`` to ``CMIP6`` and runs the recipe. However, ESMValTool
```

SarahAlidoost commented on _episodes/06-debugging.md:
````diff
@@ -300,20 +300,28 @@ esmvalcore._task.DiagnosticError: Cannot execute script 'diag_scripts/examples/d
-``path_to_esmvaltool/diag_scripts/ocean/``. Alternatively, the script path can
-be an absolute path. To examine this, we change the script path and run the
-recipe:
+``<path_to_esmvaltool>/diag_scripts/ocean/``. Alternatively, the script path can
+be an absolute path. To examine this, we create a small python script `hello.py`:
+
+```python
+with open('hello_world.txt', 'w') as f:
+    print('Fello from ESMValTool!', file=f)
+```
+
+Change and run the recipe:
````
Nice example. However, this section aims to show the difference between the absolute and relative path of the diagnostic script. "creating your own diagnostic" is another episode in the advanced part of the tutorial that would be developed in the near future.

stefsmeets replied:
It was the only thing I could get to work with my limited knowledge 3 weeks ago 😅

The absolute path you are referring to here is very deep in miniconda, so I didn't think the instruction was very useful. The path for me is nested somewhere in: `~/miniconda3/envs/esmvaltool/lib/python3.8/site-packages/`. I think it is bad practice to recommend people to go looking around there. What do you think about adding a link to the diag script from the ESMValTool directory and running them from the working directory?

SarahAlidoost commented on _episodes/06-debugging.md:
```diff
@@ -300,20 +300,28 @@ esmvalcore._task.DiagnosticError: Cannot execute script 'diag_scripts/examples/d
 40        scripts:
 41          timeseries_diag:
-42            script: path_to_esmvaltool/diag_scripts/ocean/diagnostic_timeseries.py
+42            script: ~/esmvaltool_tutorial/hello.py
```
```suggestion
42            script: <path_to_esmvaltool>/diag_scripts/ocean/diagnostic_timeseries.py
```

SarahAlidoost commented on _episodes/06-debugging.md:
```diff
@@ -300,20 +300,28 @@ esmvalcore._task.DiagnosticError: Cannot execute script 'diag_scripts/examples/d
   esmvaltool run recipe_example.yml
 ~~~
 
+Now examine `./esmvaltool_output/recipe_example_#_#/run/diag_timeseries_temperature/timeseries_diag/` to see if it worked! Of course, this is a simple example, but it shows how you can customize diagnostic scripts or even make your own. Have a look through the scripts available in the [ESMValTool repository](https://github.com/ESMValGroup/ESMValTool/blob/master/esmvaltool/diag_scripts/).
+
```
```suggestion
Now examine `./esmvaltool_output/recipe_example_#_#/run/diag_timeseries_temperature/timeseries_diag/` to see if it worked! Of course, this is a simple example, but it shows how you can customize diagnostic scripts or even make your own.

```

SarahAlidoost replied:
"creating your own diagnostic" is another episode in the advanced part of the tutorial that would be developed in the near future. Also, "Available recipe and diagnostic scripts" are already mentioned in the next section.

SarahAlidoost reviewed (changes requested):
@stefsmeets thank you for the improvements. Please see my comments.

stefsmeets commented:
I addressed your comments. There is just one part that I don't find very satisfying. Please have a look at my comment here:
https://github.com/ESMValGroup/ESMValTool_Tutorial/pull/125#discussion_r483464828

I reverted my changes, but I think this part needs some more thought.

SarahAlidoost reviewed (approved):
@stefsmeets thank you.
